book/content/boundary_conditions/concentration.md

```diff
@@ -0,0 +1,169 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.18.1
+kernelspec:
+  display_name: festim-workshop
+  language: python
+  name: python3
+---
+
+# Concentration #
```

Comment:
I think instead of Concentration + Particle Flux + Surface reactions, we can have everything under "Hydrogen Transport"?